### PR TITLE
implements and updates detail view

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:
- Image on the left (with focal point) and details on the right (box)
- Followed by normal django fields
- also consider file upload (not just image)
- there are some template django variables missing. See screenshot. Will create a ticket for Kirill.
  screen shot 2015-12-01 at 09 08 26

| BEFORE | AFTER |
| --- | --- |
| screen shot 2015-12-01 at 09 40 14 | screen shot 2015-12-01 at 09 39 54 |
| screen shot 2015-12-01 at 10 42 17 | screen shot 2015-12-01 at 10 40 30 |

### CODE REVIEW

File: filer/templates/admin/filer/image/change_form.html [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+    {# upload stuff #}
+    <script src="{% static 'filer/js/libs/class.min.js' %}"></script>
+    <script src="{% static 'filer/js/libs/jquery.min.js' %}"></script>
+    <script src="{% static 'filer/js/libs/jquery-ui.min.js' %}"></script>
+    <script src="{% static 'filer/js/addons/jquery.cookie.min.js' %}"></script>
+    <script src="{% static 'filer/js/addons/fileuploader.min.js' %}"></script>
+    <script src="{% static 'filer/js/libs/retina.min.js' %}"></script>
+    <script src="{% static 'admin/js/admin/RelatedObjectLookups.js' %}"></script>
+    <script src="{% static 'filer/js/addons/popup_handling.js' %}"></script>
+
```
Comment (vxsx) [inline filer/templates/admin/filer/image/change_form.html]: why empty line?

File: filer/templates/admin/filer/tools/image_info.html [hunk 2]
```diff
@@ -1,150 +1,67 @@
...
+                            <div class="col-xs-3">Created</div>
+                            <div class="col-xs-9">{{ original.uploaded_at }}</div>
+                        </li>
+                    {% endif %}
+                </ul>
+            {% endif %}
+            <div class="image-actions">
+                <ul class="actions-list">
+                    <li>
+                        <a href="#">
```
Comment (vxsx) [inline filer/templates/admin/filer/tools/image_info.html]: do you need a `js-` class here? or href?

Comment (lory87) [inline filer/templates/admin/filer/tools/image_info.html]: see comment with screenshot. This is still missing.

**Comment by vxsx** [comment]
overall :+1:

File: filer/templates/admin/filer/tools/image_info.html (hunk 2 above)
Comment (vxsx) [inline filer/templates/admin/filer/tools/image_info.html]: ah ok

**Comment by vxsx** [comment]
LGTM

File: filer/templates/admin/filer/tools/image_info.html [hunk 3]
```diff
@@ -1,5 +1,73 @@
 {% load i18n %}
+{% if detail %}
+<section class="image-info detail">
```
Comment (Flight) [inline filer/templates/admin/filer/tools/image_info.html]: Problems with indents

File: filer/templates/admin/filer/tools/image_info.html [hunk 4]
```diff
@@ -1,5 +1,73 @@
...
+                            <span class="icon fa fa-pencil"></span>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        {% endif %}
+        </div>
+    </div>
+</section>
+{% else %}
```
Comment (Flight) [inline filer/templates/admin/filer/tools/image_info.html:68]: Same here

**Comment by lory87** [comment]
@Flight is it no better to have the {% indentation like I have because the tag is not rendered and then I have a weird indentation on html

**Comment by Flight** [comment]
Don't worry about this in the generated HTML

**Comment by Flight** [comment]
LGTM

**Pull request merged**